login.php

```diff
@@ -0,0 +1,98 @@
+                            <label for="logincookie">Remember me for 7 days</label>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <button type="submit" class="btn btn-primary form-control"><i class="fas fa-sign-in-alt"></i>&nbsp;&nbsp;&nbsp;Log in</button>
+                    </div>
+                </form>
+                <br>
+                <div class="row">
+                    <div class="col-xs-12">
+                        <div class="box box-<?php if (!$wrongpassword) { ?>info collapsed-box<?php } else { ?>danger<?php }?>">
+                            <div class="box-header with-border pointer no-user-select" data-widget="collapse">
+                                <h3 class="box-title">Forgot password?</h3>
+                                <div class="box-tools pull-right">
+                                    <button type="button" class="btn btn-box-tool">
+                                        <i class="fa <?php if ($wrongpassword) { ?>fa-minus<?php } else { ?>fa-plus<?php } ?>"></i>
+                                    </button>
+                                </div>
+                            </div>
+                            <div class="box-body">
+                                <p>After installing Pi-hole for the first time, a password is generated and displayed
+                                    to the user. The password cannot be retrieved later on, but it is possible to set
+                                    a new password (or explicitly disable the password by setting an empty password)
+                                    using the command
+                                </p>
+                                <pre>sudo pihole -a -p</pre>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!-- /.login-card-body -->
+            <div class="login-footer" style="margin-top: 15px; display: flex; justify-content: space-between;">
+                <a class="btn btn-default btn-sm" role="button" href="https://docs.pi-hole.net/" target="_blank"><i class="fas fa-question-circle"></i> Documentation</a>
+                <a class="btn btn-default btn-sm" role="button" href="https://github.com/pi-hole/" target="_blank"><i class="fab fa-github"></i> Github</a>
+                <a class="btn btn-default btn-sm" role="button" href="https://discourse.pi-hole.net/" target="_blank"><i class="fab fa-discourse"></i> Pi-hole Discourse</a>
+            </div>
+        </div>
+    </section>
+</div>
+
+<div class="login-donate">
+    <div class="text-center" style="font-size:125%">
+        <strong><a href="https://pi-hole.net/donate/" rel="noopener" target="_blank"><i class="fa fa-heart text-red"></i> Donate</a></strong> if you found this useful.
+    </div>
+</div>
+<script src="scripts/pi-hole/js/footer.js?v=<?php echo $cacheVer; ?>"></script>
```

Comment:
Do we really need the footer script on the login page?

![Bildschirmfoto zu 2022-08-12 09-02-57](https://user-images.githubusercontent.com/26622301/184302392-43e16f4a-badd-44d2-a8a0-8992cc9ffdf7.png)

I removed it and login still works..

Reply:
The only "needed" function is the one for the checkbox style (icheck).

![image](https://user-images.githubusercontent.com/1385443/184409268-bf0f34c2-5647-4336-bb07-0bdb97cdc30e.png)

Reply:
Ah ok, then we should keep it.

The error is about: 
https://github.com/pi-hole/AdminLTE/blob/3daca13bd64db0377e7399634ddc5264626a9728/scripts/pi-hole/js/footer.js#L260

Reply:
Because `sessionTimerCounter` is null.

I will add a test to avoid this error.

Reply:
Now it complains here: https://github.com/pi-hole/AdminLTE/blob/1714b082c3a233ea3a0c344b50352c59818676fc/scripts/pi-hole/js/footer.js#L288

> Cannot read properties of null (reading 'style')

Reply:
Fixed.